docs/source/getting-started/quickstart.rst

```diff
@@ -97,9 +97,20 @@ Let's apply Laplace noise to a value.
             :end-before: /demo
 
 This is obviously not the easiest way to add noise to a number,
-but it demonstrates a number of OpenDP patterns:
+but it demonstrates a number of low-level OpenDP patterns:
 
 * Defining your metric space with ``space_of`` in Python's Context API, or a (domain, distance) tuple in any language..
 * Chaining operators together with ``>>`` in Python and Rust, or ``|>`` in R.
 * Constructing a ``Measurement`` function on your metric space with ``then_laplace``.
 * Invoking that measurement on a value to get a DP release.
```

Comment:
Since these are a sequence of steps, it seems weird to move some but not all of this? I think I like it outside because the main point is to emphasize the similarities across languages.